mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -968,6 +973,10 @@ def test_crop():
         raw1 = raw.copy().crop(tmin=tmin, tmax=tmin + 2)
         assert raw1[:][0].shape == (1, 2001)
 
+    # degenerate
+    with pytest.raises(ValueError, match='No samples.*when include_tmax=Fals'):
```

Comment:
```suggestion
    with pytest.raises(ValueError, match='No samples.*when include_tmax=False'):
```

Reply:
Causes a PEP8 error (line length)

I usually just truncate them wherever necessary to get to 80 char

Reply:
LOL. Feel free to merge then when CIs are green